lib/Minz/Migrator.php

```diff
@@ -247,11 +245,11 @@ public function upToDate() {
 	 * considered as successful. It is considered as good practice to return
 	 * true on success though.
 	 *
-	 * @return array Return the results of each executed migration. If an
+	 * @return array<string|bool> Return the results of each executed migration. If an
```

Comment:
I believe this might be the following (to be double-checked)

```suggestion
	 * @return array<string|true> Return the results of each executed migration. If an
```